src/neuroglancer/volume_rendering/volume_render_layer.ts

```diff
@@ -396,6 +460,9 @@ void main() {
               chunkTransform: {channelToChunkDimensionIndices}
             } = transformedSource;
             const {} = transformedSource;
+            const normChunkNumber = chunkNumber / chunks.size;
+            gl.uniform1f(shader.uniform('uChunkNumber'), normChunkNumber);
```

Comment:
could condition calling this on wireFrame mode

Reply:
Great spot, sorry I missed that one - will fix!